### DISCUSSION TITLE
Curso Acesso a Dados - Executando Procedures: Não entendi como a minha aplicação consegue encontrar  a procedure a ser executada

### DISCUSSION THREAD

**Discussion by oimarcella** [balta-io/forum#548] [Tira Dúvidas] [answered]
Tenho minha query escrita, fiz pelo Azure Data Studio, no entanto não entendo como minha aplicação lá no VS Code consegue saber qual a query tem na minha procedure, como é feita essa ligação?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @mlasa, como vai?

Você pode colocar o código aqui por favor? Assim fica mais fácil para eu entender e te explicar...
Coloca o código da sua procedure e o trecho que faz a chamada dela no C#.
